src/rluni/robot/RWIP.py

```diff
@@ -76,17 +78,19 @@ def __init__(
 
     def _load_config(self, config_file):
         """
-        Private method to load and parse the RWIP configuration file.
+        Private method to load and parse the robot configuration file.
         """
-        # Load the RWIP configuration file (use the default if none provided)
+        # Load the robot configuration file (use the default if none provided)
```

Comment:
Is the idea here that we want to use this file for the full robot as well? Or separate them as discussed?

Reply:
This config loader can be frozen for the rwip and the robot one may diverge. Better to have different config fields and parsers for different robots in the long term